### DISCUSSION TITLE
How to ensure I'm working in 64-bit?

### DISCUSSION THREAD

**Discussion by amunozj** [Joshuaalbert/jaxns#153] [Q&A] [answered]
Hello! I'm getting this termination condition:

`All live-points are on a single plateau (potential numerical errors, consider 64-bit)`

Does anyone know how to address this recommendation?

**Comment by Joshuaalbert** [comment, marked as answer]
Put the following at the top of your script above all other imports.

```python
from jax.config import config
config.update("jax_enable_x64", True)
```

The reason you should put it at the top is because some JAX-based packages inadvertently rely on default dtype behaviour.

**Comment by Joshuaalbert** [comment]
By the way, that termination condition can be totally expected for likelihoods that have plateaus. In these likelihood cases the only way to determine if there is something hidding inside the plateau (like a spike of posterior mass) is to increase `c`, i.e. the number of parallel sample trees. You can try setting it to `c=model.U_ndims*100`.

**Reply by amunozj** [reply]
Thank you so much for both your answer and your work Joshua!  You are the best!  I appreciate jaxns very, very much!
